test/functional/helpers/getAdobeIOAccessToken.js

```diff
@@ -9,21 +9,65 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import auth from "@adobe/jwt-auth";
-import adobeIOClientCredentials from "./adobeIOClientCredentials";
+const fs = require("fs");
 
-let accessToken;
+const auth = (...args) =>
+  // eslint-disable-next-line import/no-unresolved
+  import("@adobe/auth-token").then(({ auth: adobeAuth }) => adobeAuth(...args));
 
-/**
- * Retrieves an access token for the Adobe I/O integration used for
- * running end-to-end tests.
- * @returns {Promise<string>}
- */
-export default async () => {
-  if (accessToken) {
-    return accessToken;
+module.exports = ({ clientId, clientSecret }) => {
+  const tokenPath = ".token-cache.json";
```

Comment:
`.token-cache.json` should be added to the `.gitignore` so no one accidentally commits it.